__DOC_EXAMPLES__/annotatedExamples/resize.md

```diff
@@ -33,7 +34,7 @@ const myImage = cld.image('sample');
 // Lets give resize() a ResizeScale paramter to scale our image
```

Comment:
should we change this from ResizeScale?

Reply:
I'll it to the docs team to decide, but I tried to make it clear that it's a separate type of resizing,
they'll decide if to keep it or not though.